public/content/lessons/2019/cramers-rule/index.mdx

```diff
@@ -7,4 +7,449 @@ video: jBsC34PxzoM
 source: _2018/cramer.py
 credits:
 - Lesson by Grant Sanderson
+- Text adaptation by Kurt Bruns
+- Text adaptation by James Schloss
 ---
+
+> Jerry: Ah, you're crazy!
+>
+> Kramer: Am I? Or am I so sane that you just blew your mind?
+>
+> Jerry: It's impossible!
+>
+> Kramer: Is it?! Or is it so possible your head is spinning like a top?
+
+In a previous chapter, we talked about linear systems of equations, and sort of brushed aside the discussion of actually computing solutions to these systems. While it's true that number-crunching is something we typically leave to the computers, digging into some of these computational methods is a good litmus test for whether or not you actually understand what's going on, since this is really where the rubber meets the road.
```

Comment:
```suggestion
In a previous chapter, we talked about linear systems of equations, and sort of brushed aside the discussion of actually computing solutions to these systems. While it's true that number-crunching is something we typically leave to the computers, digging into some of these computational methods is a good test for whether or not you understand what's going on since this is where the rubber meets the road.
```